mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -1259,7 +1264,8 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT <p>This is saved on the {@code Invulnerable} NBT key.
 		COMMENT
 		COMMENT @implNote Invulnerable entities are immune from all damages except {@link
-		COMMENT DamageSource#OUT_OF_WORLD} and damages by creative mode players by default.
+		COMMENT net.minecraft.entity.damage.DamageSources#outOfWorld}
```

Comment:
Same as above, reference the type not the source

Reply:
```suggestion
		COMMENT net.minecraft.entity.damage.DamageTypes#OUT_OF_WORLD}
```